scripts/env.sh

```diff
@@ -1,3 +1,3 @@
-export CM_SCALA_VERSION=3.0.0-RC3-bin-COMMUNITY
+export CM_SCALA_VERSION=3.0.0-RC3-bin-COMMUNITY-SNAPSHOT
```

Comment:
It looks like, as dotty repository has moved forward, trying to build everything with `3.0.0-RC3-x` now won't work if the project we try to build (munit in this case) depends on has any dependency that we don't build ourselves. To fix this you might try bumping the version to `3.0.1-RC1-bin-COMMUNITY-SNAPSHOT` and/or changing the project tested in the demo to something without dependencies (I assume e.g. `squants` should work or anything else you can find)